modules/nodes-containers-init-creating.adoc

```diff
@@ -10,7 +10,9 @@ The following example outlines a simple pod which has two Init Containers. The f
 
 .Procedure
 
-. Create a YAML file for the Init Container:
+. Crate the pod for the Init Container:
```

Comment:
"Crate" should be "Create." Also, make sure additional steps end in a colon, per IBM style guide: https://www.ibm.com/docs/en/ibm-style?topic=format-procedures